modules/analyzers/spdx-analyzer/spdxanalyzer/__main__.py

```diff
@@ -86,35 +99,50 @@ def _process_filenodes(self):
                     "File {} not found in SPDX document".format(node.path))
                 continue
             spdx_doc_file_info = filtered_files[0]
-            if not isinstance(spdx_doc_file_info.conc_lics, License):
-                continue
-
-            data_nodes = []
-            logging.info("Concluded license {}".format(
-                spdx_doc_file_info.conc_lics))
-            data_nodes.append(InfoNode.DataNode(
-                type="spdxIdentifier",
-                data=spdx_doc_file_info.conc_lics.identifier
-            ))
-            data_nodes.append(InfoNode.DataNode(
-                type="name",
-                data=spdx_doc_file_info.conc_lics.full_name
-            ))
 
-            info_node = InfoNode(
-                type="license",
-                dataNodes=data_nodes
-            )
-
-            info_nodes = []
-            info_nodes.append(InfoNodeMessage(
-                uid=node.uid,
-                token=self.token,
-                infonode=info_node))
-
-            info_iterator = _generate_iterator(info_nodes)
-
-            self.aserv.SendInfoNodes(info_iterator)
+            if isinstance(spdx_doc_file_info.conc_lics, License):
```

Comment:
I know this was here before, but is it really necessary to use isinstance? IMHO it is never good style to ask an instance about its type. And if you need to do that something is odd.